### DISCUSSION TITLE
strimzi source connector to connect Azure IOT hub

### DISCUSSION THREAD

**Discussion by rameugenie** [strimzi/strimzi-kafka-operator#5799] [General]
Hi Team,

I am testing strimzi soultion for my test environment.So i am just looking for Docker image if we have something available from strimzi which connect to my Azure iot-hub and get the data stream into my strimzi kafka.

Any help/Suggestion would be very much helpful.

Thanks&Regards,
Ram

**Comment by ppatierno** [comment]
Hi,
I don't think you can connect a Kafka instance with or without Strimzi to Azure IoT Hub directly because the only protocols that it supports are AMQP 1.0, MQTT 3.1 and HTTP, not the Kafka custom protocol for sure.
What you can do is about mirroring a Kafka instance (deployed via Strimzi) to Azure Event Hub which provides a Kafka protocol head.
The following blog posts can provide you some tips on how to do that:

https://strimzi.io/blog/2020/05/14/mirror-kafka-eventhub/ (using Mirror Maker 1 which is going to be deprecated anyway)

https://strimzi.io/blog/2020/06/09/mirror-maker-2-eventhub/ (using Mirror Maker 2)

**Comment by rameugenie** [comment]
Thank you very much @ppatierno . i am wondered ,you responded us with in fraction of minuts :)
